README.md

```diff
@@ -73,16 +70,19 @@ make install
 Run `mongocli help` for a list of available commands
 or check our [online documentation](https://docs.mongodb.com/mongocli/master/) for more details.
 
-### Getting API Keys
+### Configuring MongoCLI with Atlas
+To use `mongocli` with Atlas you'll need to run `mongocli auth login` and follow the prompted steps on your terminal.
+
```

Comment:
@gssbzn Let me know if this sentence is enough or you want me to add/remove something here